.github/workflows/docker.yml

```diff
@@ -0,0 +1,44 @@
+name: Docker Image Build
+
+on:
+  push:
+    branches:
+      - 'master'
+
+    tags:
+      - '*'
+
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  Docker_Linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        llvm: ["11"]
+        ubuntu: ["18.04", "20.04"]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Build LLVM ${{ matrix.llvm }} on ${{ matrix.ubuntu }}
+      run: |
+        docker build . -t docker.pkg.github.com/lifting-bits/anvill/anvill-llvm${{ matrix.llvm }}-ubuntu${{ matrix.ubuntu }}-amd64:latest -f Dockerfile --target=dist --build-arg UBUNTU_VERSION=${{ matrix.ubuntu }} --build-arg ARCH=amd64 --build-arg LLVM_VERSION=${{ matrix.llvm }}
+    - name: Test Docker image
+      run: |
+        docker run --rm -v $(pwd):/workspace -w /workspace docker.pkg.github.com/lifting-bits/anvill/anvill-llvm${{ matrix.llvm }}-ubuntu${{ matrix.ubuntu }}-amd64:latest -spec /workspace/tools/decompile-json/tests/specs/ret0.json -bc_out /tmp/ret0.bc -ir_out /tmp/ret0.ir
```

Comment:
These tests could be run from within the container when building anvill, instead of repeating them here

Reply:
I think this is to sanity check that the built image works, entry point and all, and is not meant as a replacement for anvill tests.

Reply:
Yeah, this is to sanity check that the multi-stage Docker image actually has everything needed to run our installed artifacts....

Thinking more. It would be a nice test to install the packaged deb file that we build, and then do these sanity checks. Right now we just copy the `/opt/trailofbits` directory 🤔